streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/ProcessorParameters.java

```diff
@@ -28,17 +28,17 @@
  */
 public class ProcessorParameters<K, V> {
 
-    private final ProcessorSupplier<K, V> processorSupplier;
+    private final ProcessorSupplier<? super K, ? super V> processorSupplier;
```

Comment:
This is to be aligned with KIP-101: found we need to relax it in order to add back the strong typing and remove `unchecked` annotations.